src/java.base/share/classes/java/io/ObjectInputFilter.java

```diff
@@ -523,10 +522,15 @@ static ObjectInputFilter rejectUndecidedClass(ObjectInputFilter filter) {
      * {@systemProperty jdk.serialFilter}, its value is used to configure the filter.
      * If the system property is not defined, and the {@link java.security.Security} property
      * {@code jdk.serialFilter} is defined then it is used to configure the filter.
-     * The filter is created as if {@link #createFilter(String) createFilter} is called;
-     * if the filter string is invalid, an {@link ExceptionInInitializerError} is thrown.
-     * Otherwise, the filter is not configured during initialization and
-     * can be set with {@link #setSerialFilter(ObjectInputFilter) Config.setSerialFilter}.
+     * The filter is created as if {@link #createFilter(String) createFilter} is called,
+     * if the filter string is invalid the initialization fails and subsequent attempts to
+     * {@linkplain Config#getSerialFilter() get the filter}, {@link Config#setSerialFilter set a filter},
+     * or create an {@link ObjectInputStream#ObjectInputStream(InputStream) ObjectInputStream}
+     * throw {@link IllegalStateException}. Deserialization is not possible with an
+     * invalid serial filter.
+     * If the system property {@code jdk.serialFilter} or the {@link java.security.Security}
+     * property is not set the filter can be set with
```

Comment:
> or the {@link java.security.Security} property is not set the filter can be set ...

Is it intentional that the name of security property is left out here? Perhaps this should be:
 `or the {@link java.security.Security} property {@code jdk.serialFilter} is not set the filter ....`

or even:

`or the {@link java.security.Security} property of the same name is not set the filter....`

Reply:
Yes, for consistency, the first suggestion.